src/stickyfill.js

```diff
@@ -6,9 +6,13 @@
  *    of the polyfill, but the API will remain functional to avoid breaking things.
  */
 let seppuku = false;
+let isInitialized = false;
+
+// Check for existence of window in case this is being imported on the server in an SSR app
+const isWindowDefined = typeof window !== 'undefined';
 
 // The polyfill cant’t function properly without `getComputedStyle`.
-if (!window.getComputedStyle) seppuku = true;
+if (isWindowDefined && !window.getComputedStyle) seppuku = true;
```

Comment:
I’m not sure this will work as expected. Right after this check there’s a `document.createElement`, which will also fail on the server.

I’d just prefer to set the `suppuku` flag if the window is not defined:
```js
if (typeof window == 'undefined' || !window.getComputedStyle) seppuku = true;
```
Thus the polyfill just won’t work on the server (and it shouldn’t), but the API will remain functional (it shouldn’t be used on the server, but if it’s used for some reason, things won’t break).

`isWindowDefined` flag can be removed completely.